### DISCUSSION TITLE
Example of how to create validators that work with this

### DISCUSSION THREAD

**Discussion by bourbongoat** [Blazored/FluentValidation#54] [General]
It would be nice for the readme to show how the validators should be created to go along with this as well.

**Comment by chrissainty** [comment]
I'm sorry but I'm not sure what you mean? What would you like to see in the readme?

**Reply by bourbongoat** [reply]
You mention injecting validators with DI and loading assemblies with them.  Are these stand alone classes or built into the data models that are being bound to?  What would the corresponding validators for the example in the readme look like and how would they be injected?  Something like that.

**Reply by chrissainty** [reply]
I don't think that is really the job of this library to explain. That is how to configure model validators using DI with FluentValidation, if I'm understanding you correctly, which is not Blazor specific.
